samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -114,6 +115,35 @@ static void gen_onoff_get(struct bt_mesh_model *model,
 	if (bt_mesh_model_send(model, ctx, msg, NULL, NULL)) {
 		printk("Unable to send GEN_ONOFF_SRV Status response\n");
 	}
+
+	if (bt_mesh_model_elem(model)->addr == elements[0].addr) {
+		last_get_msg = get_msg;
+		get_msg = ONOFF_GET;
+	}
+}
+
+void gen_onoff_publisher(struct bt_mesh_model *model)
```

Comment:
After 2nd commit, mentioned issue has solved. This may satisfy PTS requirements (as per above mentioned 9 failed logs). Need PTS to re-run against it to reach at final conclusion.

Reply:
done !!